Trimble.Modus.Components/Controls/TMRadioButton/TMRadioButton.cs

```diff
@@ -0,0 +1,244 @@
+﻿using Trimble.Modus.Components.Contant;
+using Trimble.Modus.Components.Enums;
+
+namespace Trimble.Modus.Components
+{
+    public class TMRadioButton : ContentView, IDisposable
+    {
+        #region Fields
+
+        private readonly Label _label;
+        private readonly Image _radio;
+        private readonly int _defaultDimension = 24, _largeDimension = 32;
+        private readonly int _defaultFontSize = 14, _largeFontSize = 16;
+        private readonly TapGestureRecognizer _tapGesture = new();
+
+        #endregion
+
+        #region Public Properties
+
+        /// <summary>
+        /// Triggered when <see cref="IsSelected"/> changes.
+        /// </summary>
+        public event EventHandler<CheckedChangedEventArgs> SelectedChanged;
```

Comment:
Is the `CheckedChangedEventArgs` is from the Checkbox? Can we use a different EventArgs for RadioButton? like `SelectionChangedEventArgs`?
And also, please rename the propery name to `SelectionChanged `

Reply:
`CheckedChangedEventArgs` is the default EventArg used for RadioButton by maui. Hence using the same.
Renamed `SelectedChanged` to `SelectionChanged`.